public/scss/home.scss

```diff
@@ -0,0 +1,238 @@
+/* 88888888888888888    slider   8888888888888888888888 */
+.home {
+  .slider {
+    background-color: #fff;
+    margin-bottom: 20px;
+    padding: 15px 0 6px;
+    .slider_home {
+      position: relative;
+      margin: auto;
+      .prev,
+      .next {
+        cursor: pointer;
+        position: absolute;
+        top: 50%;
+        width: auto;
+        padding: 12px 16px;
+        margin-top: -22px;
+        color: white;
+        font-weight: 400;
+        font-size: 16px;
+        -webkit-transition: 0.6s ease;
+        -o-transition: 0.6s ease;
+        transition: 0.6s ease;
+        -webkit-user-select: none;
+        -moz-user-select: none;
+        -ms-user-select: none;
+        user-select: none;
+        background-color: rgba(0, 0, 0, 0.2);
+        text-transform: uppercase;
+      }
+      .prev:hover,
+      .next:hover {
+        background-color: rgba(0, 0, 0, 0.4);
+      }
+
+      /* Position the "next button" to the right */
+      .next {
+        right: 0;
+      }
+
+      /* On hover, add a black background color with a little bit see-through */
+      .dots {
+        position: absolute;
+        text-align: center;
+        bottom: 30px;
+        width: 100%;
+      }
+      .dot {
+        cursor: pointer;
+        height: 6px;
+        width: 6px;
+        margin: 0 8px;
+        background-color: #bbb;
+        border-radius: 50%;
+        display: inline-block;
+        -webkit-transition: background-color 0.6s ease;
+        -o-transition: background-color 0.6s ease;
+        transition: background-color 0.6s ease;
+      }
+
+      .active,
+      .dot:hover {
+        background-color: #717171;
+      }
+    }
+  }
+  section {
+    position: relative;
+    padding: 23px 0;
+    margin-bottom: 20px;
+    background-color: #fff;
+    &:nth-child(odd) {
+      .cat_img {
+        order: 1;
+      }
+    }
+    &:after {
+      content: "";
+      position: absolute;
+      left: 50%;
+      top: -33px;
+      width: 60%;
+      height: 5px;
+      background: #fff;
+      border-radius: 290px / 8px;
+      -moz-border-radius: 290px / 8px;
+      -webkit-border-radius: 290px / 8px;
+      box-shadow: 0 5px 15px rgba(0, 0, 0, 0.9);
+      -moz-box-shadow: 0 10px 17px rgba(0, 0, 0, 0.9);
+      -webkit-box-shadow: 0 5px 15px rgba(0, 0, 0, 0.9);
+      z-index: -3;
+      -webkit-transform: translate(-50%, -50%);
+      -ms-transform: translate(-50%, -50%);
+      transform: translate(-50%, -50%);
+    }
+  }
+  .flex_boxes {
```

Comment:
You should create a separate mould where you keep all flex related reusable CSS.

Reply:
Done